Music app/app/src/main/java/com/example/musicapp/Activities/PlayerActivity.java

```diff
@@ -141,99 +143,63 @@ public void run()
         prevThread.start();
     }
 
-    private void prevBtnClicked()
+    private void prevBtnSupp()
```

Comment:
Why these functions are needed?

Reply:
Because mister says that method prevBtnClicked is too long - so I split it into 2 functions!